client/src/pages/Landing/WhatOurUsersSay/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Text } from 'components'
+import Aderemi_pic from '../../../assets/svg/Aderemi.svg'
+import styles from './WhatOurUsersSay.module.scss'
+import polygon from './../../../assets/svg/Polygon1.svg'
+
+const WhatOurUsersSay = () => {
+
+    const user_feedback = [
+        {
+            name: 'Aderemirekun Ayomide',
+            group: 'Widower Association',
+            picture: Aderemi_pic,
+            feedback: '"Ajo Money has transformed the way I save money. Being part of a thrift group has made it easier to stay motivated and accountable. With Ajó Savings, I\'ve achieved my savings goals faster than ever before!"'
+        },
+        {
+            name: 'Aderemirekun Ayomide',
+            group: 'Widower Association',
+            picture: Aderemi_pic,
+            feedback: '"Ajo Money has transformed the way I save money. Being part of a thrift group has made it easier to stay motivated and accountable. With Ajó Savings, I\'ve achieved my savings goals faster than ever before!"'
+        },
+        {
+            name: 'Aderemirekun Ayomide',
+            group: 'Widower Association',
+            picture: Aderemi_pic,
+            feedback: '"Ajo Money has transformed the way I save money. Being part of a thrift group has made it easier to stay motivated and accountable. With Ajó Savings, I\'ve achieved my savings goals faster than ever before!"'
+        },
+    ]
```

Comment:
move this outside the react component, every re-render cause this to be recreated